packages/formatter-excel/src/formatter.ts

```diff
@@ -34,7 +34,7 @@ const startRow = 5;
  */
 
 export default class ExcelFormatter implements IFormatter {
-    public async format(messages: Array<Problem>, target: string) {
+    public async format(messages: Array<Problem>, target = '') {
```

Comment:
I made `target` optional to match actual call patterns in the code. Rather than changing it to `target?: string` here I defaulted to the empty string so we could actually successfully save (albeit with a weird name - FWIW I debated defaulting to `'unknown'` which we could still change to). 

Previously we'd fail further down in the file on the `target.replace` call, catch the error and give an error message that the file couldn't be saved.

Reply:
ok, nice!

Reply:
> I made `target` optional to match actual call patterns in the code. Rather than changing it to `target?: string` here I defaulted to the empty string so we could actually successfully save (albeit with a weird name - FWIW I debated defaulting to `'unknown'` which we could still change to).
> 
> Previously we'd fail further down in the file on the `target.replace` call, catch the error and give an error message that the file couldn't be saved.


Included part of ⬆️ comment. Also changed this to a `Fix:`.